devices/examples/YarpCameraBridgeArucoTestDevice/gazebo/models/aruco_marker_4x4_1000-0/materials/scripts/aruco_marker_4x4_1000-0.material

```diff
@@ -0,0 +1,13 @@
+material ArucoMarker4x4_1000_0/Marker
```

Comment:
I've noticed that the number of file in the material folder is really huge. Where did you find them? Are there available somewhere else? If yes we could fetch them from cmake and avoid to store them in this repo

Reply:
I had used aruco markers generated using the [ArucoGen online software](https://chev.me/arucogen/) to generate markers from a 4x4 dictionary for the robot experiments. So, for the simulation, I generated the same markers and used them as the material for the gazebo models, taking inspiration from https://github.com/joselusl/aruco_gazebo.

Reply:
> If yes we could fetch them from cmake and avoid to store them in this repo

We can do that.

Reply:
Where can we host that content? a separate repo? or shared storage?

Reply:
I have removed the models in this commit https://github.com/ami-iit/bipedal-locomotion-framework/pull/504/commits/9dc82b8384404c7c0153c8ce75e5e63592012f2f.

If we decide where to host them, I shall update the documentation accordingly on fetching and using them.